classes/classes/PlayerAppearance.as

```diff
@@ -57,11 +57,10 @@ package classes
 			var race:String = "human";
 			race = player.race();
 			//Discuss race
-			clearOutput();
-			displayHeader("Appearance");
-			if (race != player.startingRace)	outputText("You began your journey as a " + player.startingRace+ ", but gave that up as you explored the dangers of this realm.  ", false);
+			output.clear().header("Appearance");
+			if (race != player.startingRace)	output.text("You began your journey as a " + player.startingRace + ", but gave that up as you explored the dangers of this realm.  ");
 			//Height and race.
-			outputText("You are a " + feet_inch_and_metres(player.tallness) + " tall " + player.maleFemaleHerm() + " " + race + ", with " + player.bodyType() + ".", false);
+			output.text("You are a " + feet_inch_and_metres(player.tallness) + " tall " + player.maleFemaleHerm() + " " + race + ", with " + player.bodyType() + ".");
```

Comment:
I've used PlayerAppearance.as for my testing. I've left some lines edited to show you, how the whole thing works.